README.md

```diff
@@ -2,6 +2,8 @@
 
  [![Build Status](https://travis-ci.org/OpenSourceHelpCommunity/OpenSourceHelpCommunity.github.io.svg?branch=develop)](https://travis-ci.org/OpenSourceHelpCommunity/OpenSourceHelpCommunity.github.io)
 
+[![Coverage Status](https://coveralls.io/repos/github/OpenSourceHelpCommunity/OpenSourceHelpCommunity.github.io/badge.svg?branch=develop)](https://coveralls.io/github/OpenSourceHelpCommunity/OpenSourceHelpCommunity.github.io?branch=develop)
+
```

Comment:
Sorry @vaibhavsingh97  that was my first contribution. For the PR do i need to request another PR or just push commit and edit my existing comment to update my answer to question 2 and 3?